### DISCUSSION TITLE
Release typescript changes for DeviceManager

### DISCUSSION THREAD

**Discussion by justinr1234** [GrapesJS/grapesjs#4738] [Q&A]
Hello, I'm waiting on the typescript changes you made to the device manager as it is blocking typescript checks for `select` at the moment. When will it be released? Thanks! @artf

**Comment by justinr1234** [comment]
@artf Checking back, thanks!

**Reply by artf** [reply]
I'm not yet sure but I hope to release a new version soon (probably next week)
